src/paymentdisclosure.h

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) 2017 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef ZCASH_PAYMENTDISCLOSURE_H
+#define ZCASH_PAYMENTDISCLOSURE_H
+
+#include "uint256.h"
+#include "clientversion.h"
+#include "serialize.h"
+#include "streams.h"
+#include "version.h"
+
+// For JSOutPoint
+#include "wallet/wallet.h"
+
+#include <cstdint>
+#include <string>
+
+
+// Ensure that the two different protocol messages, payment disclosure blobs and transactions,
+// which are signed with the same key, joinSplitPrivKey, have disjoint encodings such that an
+// encoding from one context will be rejected in the other.  We know that the set of valid
+// transaction versions is currently ({1..INT32_MAX}) so we will use a negative value for
+// payment disclosure of -9411486, which in hex is 0xFF706462 and in ISO-8859-1 is ÿpdb.
```

Comment:
If you make the constant 0xFF626470 = -10328976, that will be "pdbÿ" in ISO-8859-1.

Reply:
Yep, thats what I changed it too.  I used this website to help: https://www.scadacore.com/tools/programming-calculators/online-hex-converter/